.github/actions/steam-upload/action.yaml

```diff
@@ -168,8 +168,8 @@ runs:
 
         print '::group::Prepare Windows x64 assets'
         mkdir -p steam-windows && pushd steam-windows
-        unzip ${root_dir}/(#i)obs-studio-(^*-pdbs).zip
-        rm ${root_dir}/(#i)obs-studio-*.zip
+        unzip ${root_dir}/(#i)obs-studio-*-windows.zip
```

Comment:
If my suggestion on #12226 is followed, this could be made fully explicit by using `*-windows-x64` (which also matches well with the description that this "Prepare(s) Windows x64 assets").

The step to remove all zip files should not need a change, because only Windows artefacts use `.zip` so this ensures that all of those (whether unzipped or not) are cleaned up before continuing.

Reply:
See discussion on #12226 about knock-on effects. I'll probably have to do a separate pass of changes to account for naming changes of just the x64 artifacts/assets.